src/core/modules/Star.hpp

```diff
@@ -178,7 +178,7 @@ struct Star
                                  double & RV,
                                  float    dyrs) const
    {
-      if (getX2() != 0. || getDx2() != 0.) {
+      if (getPreciseAstrometricFlag()) {
```

Comment:
Unit test is fails now

Reply:
🥲 I did not think it throught and that is actually a wrong logic. Have reverted the commit back. But still need to solve the issue if a stars is `Star1` having no parallax but with radial velocity in the catalog will result in nan/inf in info string.